### PR TITLE
Use typing.NamedTuple for WSMessage

### PR DESCRIPTION
**Pull request by Gobot1234**
## What do these changes do?

Adds slightly better completions for WSMessage

## Are there changes in behavior for the user?

Slightly better completions

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names.
- [x] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

**Comment by Gobot1234** [comment]
Should the https://github.com/aio-libs/aiohttp/blob/master/CHANGES/README.rst be updated, it says you should use .rst as the news entry file type?

**Comment by Dreamsorcerer** [comment]
I actually already have this change in my PR:
https://github.com/aio-libs/aiohttp/pull/6555/files#diff-29db742fa68fd59f39e1ca70379c4d478101a8ed9af8895c8fc7604bbf1b4b11

But, thanks anyway.

File: aiohttp/http_websocket.py [hunk 1]
```diff
@@ -78,10 +88,11 @@ class WSMsgType(IntEnum):
...
 
 
-_WSMessageBase = collections.namedtuple("_WSMessageBase", ["type", "data", "extra"])
-
+class WSMessage(NamedTuple):
+    type: WSMsgType
+    data: Any
+    extra: str
 
-class WSMessage(_WSMessageBase):
```
Comment (Dreamsorcerer) [inline aiohttp/http_websocket.py:84]: Good idea on dropping the intermediate class though, have updated my PR with that.

**Comment by Gobot1234** [comment]
Oh sorry I didn't realise

**Pull request closed without merging**